compiler-plugin-tests/src/test/resources/ballerina_sources/generator_tests/24_SDL_file_generation_with_multiple_services/service.bal

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/graphql;
+
+public type Person record {|
+    string name;
+    int age;
+|};
+
+service on new graphql:Listener(4000) {
+    resource function get profile() returns Person {
+        return {
+            name: "Walter White",
+            age: 52
+        };
+    }
+}
+
+service on new graphql:Listener(4000) {
+    resource function get school() returns School {
+        return new;
+    }
+}
+
+service on new graphql:Listener(4000) {
+    resource function get name() returns string {
+        return "Ballerina";
+    }
+
+    resource function get age() returns int {
+       return 27;
+    }
+}
```

Comment:
Multiple services using the same port and the same base path is not a valid use case. Shall we use different ports (case 1), and same port with different paths (case 2) as tests?